src/node/internal/crypto.d.ts

```diff
@@ -44,6 +44,11 @@ export function createSecretKey(key: ArrayBuffer | ArrayBufferView): CryptoKey;
 export function createPrivateKey(key: InnerCreateAsymmetricKeyOptions): CryptoKey;
 export function createPublicKey(key: InnerCreateAsymmetricKeyOptions): CryptoKey;
 
+// Spkac
+export function verifySpkac(input: ArrayBufferView|ArrayBuffer): boolean;
```

Comment:
The Node.js docs indicate that exportPublicKey and exportChallenge return a Buffer – is that equivalent to `null | ArrayBuffer` here? All three functions also need to support string and Buffer inputs.

Reply:
These are the types for the internal call. The `Buffer` and `string` handling should be taken care of in the ts file that defines the public API.